project/NativeImage.scala

```diff
@@ -202,15 +195,16 @@ object NativeImage {
 
       var args: Seq[String] =
         Seq("-cp", cpStr) ++
-        quickBuildOption ++
-        debugParameters ++ staticParameters ++ configs ++
+        staticParameters ++
+        configs ++
         Seq("--no-fallback", "--no-server") ++
         Seq("-march=compatibility") ++
         initializeAtBuildtimeOptions ++
         initializeAtRuntimeOptions ++
         buildMemoryLimitOptions ++
         runtimeMemoryOptions ++
         additionalOptions ++
+        additionalOpts.value ++
```

Comment:
`additionalOptions`.. `additionalOpts`.. `additionalFoo`. This is getting confusing.

Reply:
A bit. One is a parameter to the method, and the other one is a value from the task. I don't know now how to merge these two, with minimal changes.